diff -Nru synergy-1.4.17-Source/src/CMakeLists.txt synergy-1.4.17-Source-modified-gtest/src/CMakeLists.txt
--- synergy-1.4.17-Source/src/CMakeLists.txt	2014-02-26 18:46:02.000000000 +0100
+++ synergy-1.4.17-Source-modified-gtest/src/CMakeLists.txt	2014-03-27 16:44:39.476757747 +0100
@@ -18,7 +18,3 @@
 add_subdirectory(cmd)
 add_subdirectory(plugin)
 add_subdirectory(micro)
-
-if (NOT ${CMAKE_SYSTEM_NAME} MATCHES "IRIX")
-	add_subdirectory(test)
-endif()
